index.bs

```diff
@@ -5237,30 +5237,30 @@ partial interface MLGraphBuilder {
     : <dfn>scales</dfn>
     ::
         A list of length 2.
-        Specifies the scaling factor in each spatial dimensions of the input: *[scaleHeight, scaleWidth]*.
+        Specifies the scaling factor in each dimensions of the input in {{MLResample2dOptions/axes}} : *[scaleForFirstAxis, scaleForSecondAxis]*.
```

Comment:
Plurality agreement (existing).
```suggestion
        Specifies the scaling factor in each dimension of the input in {{MLResample2dOptions/axes}} : *[scaleForFirstAxis, scaleForSecondAxis]*.
```

Slight wording suggestion to avoid "in" twice in a row. That also matches the wording below "Specifies the target sizes for each dimension..." more closely:
```suggestion
        Specifies the scaling factor for each input dimension from {{MLResample2dOptions/axes}} : *[scaleForFirstAxis, scaleForSecondAxis]*.
```